pkg/machinery/config/encoder/markdown.go

```diff
@@ -195,9 +158,18 @@ func encodeYaml(in interface{}, name string) string {
 		lines[i] = strings.TrimRight(line, " ")
 	}
 
-	return fmt.Sprintf("``` yaml\n%s```", strings.Join(lines, "\n"))
+	return fmt.Sprintf("{{< highlight yaml >}}\n%s{{< /highlight >}}", strings.Join(lines, "\n"))
```

Comment:
A possible issue here is that this is coupling the outputted markdown file with Hugo's rendering engine.